src/drivers/bootloaders/src/sched/timer.c

```diff
@@ -321,7 +321,7 @@ bl_timer_id timer_allocate(bl_timer_modes_t mode, time_ms_t msfromnow,
 
```

Comment:
@Inspirati  Please branch nuttx_v3 and Cherry pick this commit in to you new branch.